west.yml

```
@@ -112,6 +116,14 @@ manifest:
       repo-path: mbedtls
       revision: mbedtls-2.16.6
       remote: armmbed
+    - name: nanopb
+      path: nanopb
```

based on https://github.com/nanopb/nanopb/blob/master/README.md this seems like a better location:
```suggestion
      path: modules/lib/nanopb
```